pkg/apis/workloads/v1alpha1/console_types.go

```diff
@@ -120,6 +120,37 @@ type ConsoleTemplateSpec struct {
 	// +kubebuilder:validation:Minimum=0
 	// +kubebuilder:validation:Maximum=604800
 	DefaultTTLSecondsAfterFinished *int32 `json:"defaultTtlSecondsAfterFinished,omitempty"`
+
+	// List of authorisation rules to match against in order from top to bottom.
+	// +optional
+	AuthorisationRules []ConsoleAuthorisationRule `json:"authorisationRules,omitempty"`
+	// Default authorisation rule to use if no authorisationRules are defined or no authorisationRules match.
```

Comment:
```suggestion
	// Default authorisation rule to use if no authorisation rules are defined or no authorisation rules match.
```
Maybe nitpicky but I think it reads better if we don't refer to the exact field names.